config.example/inventory

```diff
@@ -53,6 +53,9 @@ kube-node
 slurm-master
 slurm-node
 
+[easy-build-node]
```

Comment:
Can we just set this to slurm-master and get rid of the easy-build-node group as a default?

Reply:
No. Sometimes we need to build the module on a GPU node. The slurm master may or may not have GPUs. I think we should give the option to have a separate node for eb. Worst case scenario we default it slurm-master.

Reply:
We dn't need to give the option.

If this is a reasonable default, let's leave it. If a user needs to run it on a different box they can just add a new group to the inventory file then update the group in the playbook.

This is what we are now doing in the nfs playbook as well.

Reply:
Agree with Adam, we do not need a new group for this, it just adds complexity. The 'hosts' parameter in the playbook should be set to 'all', and the user can run the playbook on whatever node they want using the `-l` flag with `ansible-playbook`